stubs/dateparser/dateparser/calendars/__init__.pyi

```diff
@@ -0,0 +1,20 @@
+from typing import Any
+
+from dateparser.parser import _parser
+
+class CalendarBase:
+    parser: Any = ...
+    source: Any = ...
+    def __init__(self, source: Any) -> None: ...
+    def get_date(self): ...
+
+class non_gregorian_parser(_parser):
+    calendar_converter: Any = ...
+    default_year: Any = ...
+    default_month: Any = ...
+    default_day: Any = ...
+    non_gregorian_date_cls: Any = ...
+    @classmethod
+    def to_latin(cls, source: Any): ...
+    @classmethod
+    def parse(cls, datestring: Any, settings: Any): ...
```

Comment:
We prefer leaving out `: Any` from function arguments when the actual type is not annotated yet This way, an argument like `foo: Any` means that it really accepts anything or there's no way to do better with the current type system. For example:

```suggestion
    def parse(cls, datestring, settings): ...
```

This only applies to function arguments. Attributes and global variables still need to be specified as `Any` to avoid syntax errors.

I think newer versions of stubgen don't create `: Any` arguments to functions when they are not necessary, but I'm not sure. I don't use stubgen very often.

Reply:
> We prefer leaving out : Any from function arguments when the actual type is not annotated yet

Okay!

> I don't use stubgen very often.

Any interesting alternatives to stubgen?

Reply:
stubgen from mypy 0.910 has quite a few improvements in this regard and conforms to our guidelines much better.

Reply:
>Any interesting alternatives to stubgen?

I'm not the right person to answer this, as I don't create new stubs very often. I think I have only ever used stubgen once, when I generated pygments stubs in #5759.